manifests/install.pp

```diff
@@ -215,7 +215,7 @@
         }
       }
 
-      if "${::python::version}" =~ /^3/ { #lint:ignore:only_variable_string
+      if "${::python::version}" =~ /^python3/ { #lint:ignore:only_variable_string
```

Comment:
I see the lint ignore, but wonder if there's any reason this can't / shouldn't just be 
`if $python::version =~ /^python3/`